contracts/HealthPass721.sol

```diff
@@ -11,24 +11,59 @@ contract HealthPass721 is ERC721URIStorage, Ownable {
     using Counters for Counters.Counter;
     Counters.Counter private _tokenIds;
 
+ //We can brainstorm about it
```

Comment:
Just a general question, what are the guidelines around gas based on how much data we store on-chain? Is it possible to move this to attributes which are saved on IPFS?

Reply:
True, thats a genuine thing, if we want to have vaccine certificate customisations, then structure will expand, or else some other optimisation technique like compression